src/server/proxy_test.go

```diff
@@ -33,6 +33,7 @@ import (
 	"github.com/pachyderm/pachyderm/v2/src/internal/minikubetestenv"
 	"github.com/pachyderm/pachyderm/v2/src/internal/require"
 	"github.com/pachyderm/pachyderm/v2/src/internal/testutil"
+	tu "github.com/pachyderm/pachyderm/v2/src/internal/testutil"
```

Comment:
I think this is getting imported twice.  Just call it "testutil" below and don't import the aliased version.